src/main/java/core/basesyntax/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
+    @Override
+    public void write(String report, String path) {
+        checkReportForNull(report, path);
+        try (BufferedWriter bufferedWriter =
+                     new BufferedWriter(new FileWriter(path))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write report in file((", e);
```

Comment:
what is the report file? Add more info 
```suggestion
            throw new RuntimeException("Can't write report in file", e);
```